lib/features/popup-menu/PopupMenuProvider.ts

```diff
@@ -19,6 +21,9 @@ export type PopupMenuHeaderEntry = {
   active?: boolean;
   className: string;
   id: string;
+  imageUrl?: string;
+  imageHtml?: string;
+  label?: string;
```

Comment:
Let's also cover these types in the types tests: https://github.com/bpmn-io/diagram-js/blob/develop/lib/features/popup-menu/PopupMenuProvider.spec.ts

Reply:
added tests via [ccec0a0](https://github.com/bpmn-io/diagram-js/pull/819/commits/ccec0a00c9a9cccb7d4feaf3135ba8cb19b1ede7)